Stack/Core/Security/Certificates/ICertificateValidator.cs

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 1996-2022 The OPC Foundation. All rights reserved.
+   The source code in this file is covered under a dual-license scenario:
+     - RCL: for OPC Foundation Corporate Members in good-standing
+     - GPL V2: everybody else
+   RCL license terms accompanied with this source code. See http://opcfoundation.org/License/RCL/1.00/
+   GNU General Public License as published by the Free Software Foundation;
+   version 2 of the License are accompanied with this source code. See http://opcfoundation.org/License/GPLv2
+   This source code is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*/
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace Opc.Ua
+{
+    /// <summary>
+    /// An abstract interface to the certificate validator.
+    /// </summary>
+    public interface ICertificateValidator
+    {
+        /// <summary>
+        /// Validates a certificate.
+        /// </summary>
+        void Validate(X509Certificate2 certificate);
```

Comment:
not sufe how this maps into the old UaChannel implementation with WCF

Reply:
The TcpChannel.Asymetric.cs calls ` (Quotas.CertificateValidator as Opc.Ua.CertificateValidator.WcfValidatorWrapper).Validate(senderCertificateChain);` and doesn't use this interface method